### ISSUE TITLE
Tooltip not 508 keyboard accessible

### ISSUE THREAD

**Issue by evalaidc** [ag-grid/ag-grid#3073]
I'm submitting a ... (check one with "x")

[ ] bug report => search github for a similar issue or PR before submitting
[x ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/ceolter/ag-grid/blob/master/CONTRIBUTING.md#question
Current behavior

Tooltip is not able to be forced to stay open nor keyboard accessible for 508 compliance.

Expected behavior

As we have provided tooltipField, and the tooltip appears, the ability to navigate using the keyboard for people with accessibility issues should be an option.

What is the motivation / use case for changing the behavior?
508 compliance

Please tell us about your environment:: Windows 10 enterprise, VSCode, npm, IIS

ag-Grid version: 20.2.0, latest version, also use version 20.2.0 of ag-grid-enterprise
Browser:: Chrome & IE11
Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.99 Safari/537.36
Mozilla/5.0 (Windows NT 6.1; WOW64; Trident/7.0; SLCC2; .NET CLR 2.0.50727; .NET CLR 3.5.30729; .NET CLR 3.0.30729; Media Center PC 6.0; .NET4.0C; .NET4.0E; wbx 1.0.0; rv:11.0) like Gecko
Language: [all | TypeScript X.X | ES6/7 | ES5]

**Comment by makinggoodsoftware** [comment]
Hi,

Thank you for raising this issue for us.

As I am sure you are aware, we offer two products:

ag-Grid - this is entirely free to use in line with our MIT license. (issues reported via GitHub)
ag-Grid Enterprise - our commercial version which funds the development of both versions. (issues reported via zendesk)

If you are a customer, you should be using ag-Grid Enterprise in which case you should have access to our customer support system (via zendesk), and that is the only channel where we accept feature requests.

If you are an ag-Grid Enterprise Customer, please raise this in our Zendesk support system - contact info@ag-grid.com for access. This is our primary channel for Support.

**Comment by kiril-matev** [comment]
Hello!

Thank you for bringing this up with us. 

We have added a feature request in our backlog to track this requirement and implement it in the future.

You can track it with this reference:
**AG-3523 [Tooltip] Allow custom tooltips to remain visible as long as the mouse hovers the tooltip (even if it leaves the bounds of the cell the tooltip is for) to allow interacting with tooltip content and for accessibility support**

You can follow the status of feature requests, bugs and releases using our pipeline:
https://www.ag-grid.com/ag-grid-pipeline/

**Now that this requirement is recorded in our backlog, we will soon close the GitHub issue because we will not update it when this feature is added in a future version of AG Grid. The best way to get an update on this item is to sign up for AG Grid new release notifications. Once a new AG Grid version is released, look for the item reference above (AG-XXXX) on the changelog page to see if it's resolved in that version:
https://www.ag-grid.com/ag-grid-changelog/**

Thanks again for bringing this up with us - we appreciate your time!

**Comment by seanh169** [comment]
Will this enable the tooltip to be triggered on cell focus ? (to address accessibility/keyboard navigation)
